### ISSUE TITLE
bad nuke targetting

### ISSUE THREAD

**Issue by eronoobos** [eronoobos/BABAR-The-Shardifant#60]
Nuke targetting needs a bit of help.
1. I thought nukes only target buildings, but actually they will try to nuke a lone unit. 
2. Nukes its own units a lot. Should make sure where it's attacking doesn't have friendlies near it.

**Comment by pandaro** [comment]
or give a fake target to all mobile units when a nuke is on fire, that give the order to move away

**Comment by eronoobos** [comment]
good point, although the order should be given a little bit before (15 seconds?) firing.

point 1 would remain a problem (it's a waste of a nuke)
